app/database_etl/airtable_records_handler/airtable_records_getter.py

```diff
@@ -76,11 +100,39 @@ def get_all_records():
     # If request was not successful, there will be no records field in response
     # Just return what is in cached layer and log an error
     except KeyError as e:
-        body = "Results were not successfully retrieved from Airtable API. " \
-               "Please check connection parameters in config.py and fields in airtable_fields_config.json."
-        request_info = {
-            "url": url,
-            "headers": json.dumps(headers)
-        }
-        send_api_error_slack_notif(body, data, error=e, request_info=request_info, channel='#dev-logging-etl')
-        return request_info
+        _handle_airtable_error(e, data, url)
+
+'''
+Updates population_group_options table in whiteclaw with entries from Sample Frame GOI table in Serosurveillance Base in Airtable (name, french translation and order).
+These records will be used for the "Population group" filters on serotracker.com.
+'''
+def ingest_sample_frame_goi_filter_options():
+    data = _airtable_get_request(AIRTABLE_SAMPLE_FRAME_GOI_OPTIONS_REQUEST_URL)
+    # Try to get records from data if the request was successful
+    try:
+        # Get sorted records
+        records = data["records"]
+
+        current_time = datetime.now()
+
+        # Add records to table 
+        with db_session() as session:
+            records_to_add = [PopulationGroupOptions(id=uuid4(), 
+                                                    name=record['fields']['Name'], 
+                                                    french_name=record['fields']['French Name'] if 'French Name' in record['fields'] else None, 
+                                                    order=record['fields']['Order'] if 'Order' in record['fields'] else None, 
+                                                    created_at=current_time) 
+                             for record in records if record['fields']]
+            session.bulk_save_objects(records_to_add)
+            session.commit()  
+        return 
+    except KeyError as e:
+       return _handle_airtable_error(e, data, AIRTABLE_SAMPLE_FRAME_GOI_OPTIONS_REQUEST_URL)
```

Comment:
change from ```return _handle_airtable_error(e, data, AIRTABLE_SAMPLE_FRAME_GOI_OPTIONS_REQUEST_URL)``` to ```_handle_airtable_error(e, data, AIRTABLE_SAMPLE_FRAME_GOI_OPTIONS_REQUEST_URL)
return```. we don't want to return the return value of the function because downstream nothing is consuming it so we can just ```return``` which will return null

Reply:
should i include a return after `handle_airtable_error(e, data, AIRTABLE_SAMPLE_FRAME_GOI_OPTIONS_REQUEST_URL, headers)` or should we be good?